gen-cpp/CppServer.cpp

```diff
@@ -0,0 +1,51 @@
+#include <thrift/protocol/TBinaryProtocol.h>
+#include <thrift/server/TServer.h>
+#include <thrift/server/TThreadPoolServer.h>
+#include <thrift/server/TNonblockingServer.h>
+#include <thrift/server/TThreadedServer.h>
+#include <thrift/transport/TServerSocket.h>
+#include <thrift/transport/TBufferTransports.h>
+#include <thrift/concurrency/ThreadManager.h>
+#include <thrift/concurrency/PosixThreadFactory.h>
+
+#include <iostream>
+
+#include "RacconChat.h"
+#include "CppRacconChat.h"
+
+using namespace ::apache::thrift;
+using namespace ::apache::thrift::protocol;
+using namespace ::apache::thrift::transport;
+using namespace ::apache::thrift::server;
+using namespace ::apache::thrift::concurrency;
+
+using boost::shared_ptr;
+
+using namespace ::tutorial;
+
+int32_t main(int32_t argc, char **argv) {
+  int32_t port = 9090;
```

Comment:
use google gflags and glog for printing and passing parameters https://github.com/gflags/gflags